src/plugins/vis_type_timeseries/public/application/components/vis_types/_vis_types.scss

```
@@ -2,10 +2,15 @@
   display: flex;
   flex-direction: column;
   flex: 1 1 100%;
-  overflow: auto;
+  position: relative;
 
   .tvbVisTimeSeries {
-    overflow: hidden;
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    width: 100%;
+    height: 100%;
```

Should be able to omit this `width` and `height` if using `top`, `right`, `bottom`, and `left`.

```suggestion
    top: 0;
    right: 0;
    bottom: 0;
    left: 0;
```

---

Thank you @MichaelMarcialis, you are absolutely right